### DISCUSSION TITLE
Allure_Spock2 missing @ignore/@IgnoreIf test cases at the report

### DISCUSSION THREAD

**Discussion by Guter1988** [allure-framework/allure2#1843] [Bug Reports]
Hello,

In the last weeks, Allure_Spock2 has been released.
Before this release, I used Allure_Junit5 with the Spock framework.

When I have @Ignore or @IgnoreIf on a test, the Allure report shows it as a Gray test with the comment @Ignore.
No, at the new Allure_Spock2, these cases (of @Ignore and @ignoreIf) disappeared from the report (test cases missing).
They are not written in the report at all.
So when I used the Junit5 module, they were shown as ignoring and gray tests. But when using Spock2, they are not shown at all in the report.

How can we fix it? What can be a workaround?

**Comment by iriiiina** [comment]
I have the same problem. I use `spock.lang.Ignore` for ignoring some tests and don't see them anymore in the report after updating to Allure 2.

**Comment by Guter1988** [comment]
I try to find how it is working at Junit5 to @Override same at my project

**Comment by Guter1988** [comment]
Hi @iriiiina,
For now, I have found a solution (that will work for us) and may work for you:
`Allure.getLifecycle().updateTestCase(testResult ->
                    testResult.setStatus(Status.SKIPPED))`

I wrap it in an if statement.

Cheers,
Guy

**Reply by karltiirik** [reply]
@Guter1988, can you give an example where to put the testcase result status setting. Like into your `GenericSpecification` or whatever extends `Specification` class and is the superclass of the tests?

But at what point should you hook into the lifecycle? In a `setup()`? method? And what would the if evaluation be?
